Lib/test/test_ctypes/test_c_simple_type_meta.py

```diff
@@ -160,11 +160,11 @@ def test_bad_type_message(self):
             class F(metaclass=PyCSimpleType):
                 _type_ = "\0"
         message = str(cm.exception)
-        expected_type_chars = list('cbBhHiIlLdCEFfuzZqQPXOv?g')
+        expected_type_chars = list('cbBhHiIlLdDFGfuzZqQPXOv?g')
```

Comment:
Very minor nit: the order here is a bit weird, should't it be something like `"...fFdDgG..."` or `"...fdgFDG"` ?

Reply:
It not ordered.